composer.json

```diff
@@ -19,11 +19,12 @@
         "consolidation/robo"                    : "~1"
     },
     "require-dev": {
-        "joomla-projects/joomla-testing-robo"   : "~1.0"
+        "joomla-projects/joomla-testing-robo"   : "dev-container-test"
     },
     "autoload": {
         "psr-4": {
             "Codeception\\Module\\"             : "src"
         }
-    }
+    },
+    "minimum-stability": "dev"
 }
```

Comment:
I was missing this: please also add `"prefer-stable": true` since it's only using development versions